### ISSUE TITLE
Help with a fresh unmanaged namespace .🍉

### ISSUE THREAD

**Issue by qjouda** [Strata-Labs/bns-v2-sdk#1]
We've launched a new emoji namespace on BNS-V2 named xn--ji8h, set as unmanaged and frozen with a 0 BASE price, unchangeable. After successfully completing namespace-preorder, reveal, and launch, it's active on bnsv2.com. However, name registration options (fast or two-step) are disabled despite names appearing available. How can we start claiming names in xn--ji8h? 

namespace-launch transaction: https://explorer.hiro.so/txid/0x51de15f9bdd0dda1f7361cba58f30e468824f49505b67ef32c505a939d3e71f3?chain=mainnet

Screenshots:
- for .🍉 it doesn't show buttons:
<img width="1071" alt="image" src="https://github.com/user-attachments/assets/7db61260-0d36-40e4-88f8-2976c7bedece">

- for other namespaces, works as exected
<img width="858" alt="image" src="https://github.com/user-attachments/assets/5ccdc427-5110-4f58-8003-6bda84232263">

Thanks 🍉

**Comment by Patotking12** [comment]
Hey @qjouda thanks for opening the issue, I'll look into it and get back to you latest tomorrow

**Comment by qjouda** [comment]
Hey @Patotking12  I'm providing more details after some research. I tried to use bns-v2-sdk and stacks/connect npm libraries and call the `name-claim-fast` from my local, rather than depending on `bnsv2.com` UI. 

I was successful claiming a name on different namespaces other than 🍉. When I try the following:
- name-claim-fast is failing with `(err u3) (response uint)` 
- name-preorder is failing with `(err u108) (response uint)`

This is only happening when trying to register `something.🍉`

When I called namespace-reveal, I input Zero for BASE, COEFF, and others: here is the tx-link for it: https://explorer.hiro.so/txid/0xd11a44186758d359086199558cfc4e4deb01e8f92bee3cb4c9ab7ecc1fbdc0e0?chain=mainnet

I hope we are able to use 🍉 and not lose it for ever

thank you for your help

**Comment by Patotking12** [comment]
Hey @qjouda I finished looking into it.

err u3 means that you can not transfer 0 stx err u108 means that the stx burnt was not enough.

Basically what this means is that there can not be "free" mints in unmanaged namespaces.

Now looking at how you set up the namespace... unfortunately it is lost forever since you disabled the ability to modify the price.

**Comment by qjouda** [comment]
so sad

**Comment by qjouda** [comment]
thank you for following up

**Comment by Patotking12** [comment]
Of course! If you need help with anything else or setting up a new namespace let me know
